tests/runtime/dwarf

```diff
@@ -45,6 +45,13 @@ REQUIRES_FEATURE dwarf
 TIMEOUT 5
 BEFORE ./testprogs/uprobe_test
 
+NAME uprobe inlined function
+PROG uprobe:./testprogs/inline_function:square { printf("%d\n", ++@count); }
```

Comment:
what about instead printing the function name via `printf("%s\n", usym(reg("ip")));` to confirm it's "square"?

Reply:
It is "square" for only one of the probed call, the two inlined call will resolve to "main".
I'll remove the `printf` and let bpftrace print the `@count` value at the end.

Reply:
> It is "square" for only one of the probed call, the two inlined call will resolve to "main".

Ohh that's interesting. I guess it wouldn't be correct to return `square` for `usym(reg("ip"))` for when square is inlined but when we finally get inline info in our stack traces it should definitely come up as the leaf function for a `ustack`.